io/eolearn/tests/test_processing_io.py

```diff
@@ -198,8 +202,12 @@ def test_additional_data(self):
         eopatch = task.execute(bbox=self.bbox, time_interval=self.time_interval)
 
         bands = eopatch[(FeatureType.DATA, 'BANDS')]
-        is_data = eopatch[(FeatureType.MASK, 'SCL')]
-        scl = eopatch[(FeatureType.MASK, 'IS_DATA')]
+        is_data = eopatch[(FeatureType.MASK, 'IS_DATA')]
```

Comment:
This fails for me:

```
>       self.assertTrue(is_data.dtype == np.bool)
E       AssertionError: False is not true
```

Reply:
It doesn't fail in [CI](https://travis-ci.org/github/sentinel-hub/eo-learn/jobs/706471454#L1797). Could you check your eo-learn installation and try again?

Reply:
Deleted eolearn and sentinelhub-py and reinstalled. Now it works.